submissions/TaniaBondarenko/dom_task/my_javascript.js

```diff
@@ -0,0 +1,50 @@
+const content = document.getElementsByClassName("content");
+const month = document.getElementsByClassName("month");
+const imagesPaths = ["images/jan.jpg","images/feb.jpg","images/mar.jpg",
+    "images/apr.jpg", "images/may.jpg", "images/june.jpg",
+    "images/july.jpg", "images/aug.jpg", "images/sep.jpg",
+    "images/oct.jpg", "images/nov.jpg", "images/dec.jpg"];
+ 
+const soundsPaths = ["sounds/jan.mp3", "sounds/feb.mp3", "sounds/mar.mp3",
+"sounds/apr.mp3","sounds/may.mp3","sounds/jun.mp3",
+"sounds/jul.mp3","sounds/aug.mp3","sounds/sep.mp3",
+    "sounds/oct.mp3", "sounds/nov.mp3", "sounds/dec.mp3"];    
+
+window.onload = function () {
+    content[0].style.display = "block";
+    document.querySelector('.nav').addEventListener('click', showMonth);
+    function showMonth(event) {
+        if (event.target.className == 'month') {
+            const dataMonth = event.target.getAttribute('data-month');
+            for (let i = 0; i < month.length; i++){
```

Comment:
You can simplify finding your active element using `document.querySelector('.month.active')` because you have only one active element. No need to iterate through the whole list of elements

Reply:
fixed